authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/helper/DataCapturedLanguagesHelper.java

```diff
@@ -0,0 +1,40 @@
+package io.mosip.authentication.common.service.helper;
```

Comment:
What's the difference between ComputekeyHelper and DatacapturedLanguagesHelper? can we not have one language util class and use it?